autofill.gypi

```diff
@@ -50,6 +50,7 @@
           '<(libchromiumcontent_dir)/signin_core_browser.lib',
           '<(libchromiumcontent_dir)/signin_core_common.lib',
           '<(libchromiumcontent_dir)/os_crypt.lib',
+          'crypt32.lib',
```

Comment:
@darkdh pointed out that both electron and chrome have the flag, so I think we should add it even if it isn't strictly necessary